module.nix

```
@@ -89,6 +93,21 @@ in
         description = "Evaluate locally instead of on the remote when rebuilding";
       };
 
+      deploy-method = mkOption {
+        type = types.enum [ "copy" "archive" ];
+        default = "copy";
```

I would expect more people running into the issue of not having `git` installed to download fresh inputs or trying to transfer local / private flakes than into slow connection issues. The Nix ecosystem seems already infamous for its crazy bandwidth/storage requirements 😁. So from a maintenance perspective, to get less questions from people not reading the full documentation (even myself only read the README actually), maybe it would be best to use `archive` as a default?

```suggestion
        default = "archive";
```

---

I have not tested this, but do you even need git installed? I think nix itself downloads the inputs during build on it's own

---

In that case I'll leave it with default on copy since it should be faster or equal on most setups.